sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_models.hpp

```diff
@@ -970,4 +970,29 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
     IssuerProperties Properties;
   };
 
+  /**
+   * @brief The contact information for the vault certificates.
+   *
+   */
+  struct CertificateContact
+  {
+    /**
+     * @brief Contact email address of.
```

Comment:
This doc sentence seems like a fragment and awkward to read.

What are we trying to say here?